backend/src/interests/definition/interests.youtube.control.ts

```diff
@@ -0,0 +1,9 @@
+export const INTERESTS_YOUTUBE_CONTROL = {
+  PLAY: 'IMAGE',
+  STOP: 'STOP',
+  TIMELINE: 'TIMELINE',
+  SPEED: 'SPEED',
+  DRAGGING: 'DRAGGING',
+} as const;
+
+export type interestsYoutubeControl = (typeof INTERESTS_YOUTUBE_CONTROL)[keyof typeof INTERESTS_YOUTUBE_CONTROL];
```

Comment:
```suggestion
export type interestsYoutubeControl = typeof INTERESTS_YOUTUBE_CONTROL[keyof typeof INTERESTS_YOUTUBE_CONTROL];
```

소괄호를 넣으신 이유가 따로 있으실까요?

Reply:
소괄호가 없이 작성을 했었는데 저장을하니까 소괄호가 생겨서 확인해보니까 vscode 설정으로 잡혀있어서 해당 설정 바꿔서 소괄호 강제 안하도록 했습니다. 수정해서 올릴게요